src/Application/Components/Mint/MintShop.jsx

```diff
@@ -11,18 +11,15 @@ const MintShop = React.createClass({
     let message;
     let onUpgrade;
     if (nextDenomination) {
-      message =
-        `next denomination: ${nextDenomination.id}, ` +
-        `income per click: ${nextDenomination.incomePerClick}, ` +
-        `price to unlock: ${nextDenomination.priceToUnlock}`;
+      message = `Upgrade Denomination for $${nextDenomination.priceToUnlock}`;
```

Comment:
no opinion, i put the least amount of thought in how to present the "click this to upgrade" visual treatment.  visual consistency with Real Estate menu would be nice, tho we would want to differentiate it based on differences in nature, like the single upgrade path vs. multiple things to buy notion if we're sticking with single upgrade path, which i'm into at this point